src/style.css

```diff
@@ -1 +1,84 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+}
+
+body {
+  padding: 0;
+}
+
+.search-bar {
+  margin: 20px 0;
+}
+
+.search-bar__input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  font-family: Avenir, sans-serif;
```

Comment:
Don't fix container size (if there is no such a requirement). In this case, setting a fixed height for the input element might lead to content overflow or clipping if the user changes their browser's default font size. Consider using `min-height` instead of `height` to improve accessibility.